project-base/src/Shopsys/ShopBundle/Controller/Front/CartController.php

```diff
@@ -90,14 +90,11 @@ public function __construct(
      */
     public function indexAction(Request $request)
```

Comment:
If there are `@param` tags in docblock, there should be `@return` tag also. (applies for a whole file)

Reply:
We do not use return annotations for controller actions